mate-user-guide/C/gospanel.xml

```diff
@@ -1267,27 +1267,27 @@ See <xref linkend="panels-addobject"/> for more on this.</para>
     <para>You can add the following types of menu to your panels: </para>
     <itemizedlist>
       <listitem>
-        <para><guimenu>Menu Bar</guimenu>: You can access almost
+        <para><guimenu>Classic Menu</guimenu>: You can access almost
 all the standard applications, commands, and configuration options from
-the menus in the Menu Bar. It contains the <guimenu>Applications</guimenu>, 
+the menus in the Classic Menu. It contains the <guimenu>Applications</guimenu>, 
 <guimenu>Places</guimenu>, and <guimenu>System</guimenu> menus.</para>
-        <para>To add a <guimenu>Menu Bar</guimenu> to a panel, right-click
+        <para>To add a <guimenu>Classic Menu</guimenu> to a panel, right-click
 on any vacant space on the panel. Choose <guimenu>Add to Panel</guimenu>, then 
-choose <application>Menu Bar</application> from the Add to Panel dialog. 
+choose <application>Classic Menu</application> from the Add to Panel dialog. 
 See <xref linkend="panels-addobject"/> for more on this.</para>
       </listitem>
       <listitem>
-        <para><guimenu>Main Menu</guimenu>: The Main Menu contains the same 
-        items as the Menu Bar, but organizes them into one menu instead of three. 
+        <para><guimenu>Compact Menu</guimenu>: The Compact Menu contains the same 
+        items as the Classic Menu, but organizes them into one menu instead of three. 
         It takes up less space on the panels as a result.</para>
-        <para>To add a <guimenu>Main Menu</guimenu> to a panel, right-click
+        <para>To add a <guimenu>Compact Menu</guimenu> to a panel, right-click
```

Comment:
Superfluous noun before Compact Menu?

Reply:
Agree that we should change "To add a Compact Menu" to "To add [the?] Compact Menu", but this is just a name change, so I wouldn't block a merge for this.